docs/developers-guide/e2e-tests.md

```diff
@@ -10,7 +10,13 @@ Metabase uses Cypress for “end-to-end testing”, that is, tests that are exec
 
 Metabase’s Cypress tests are located in the `e2e/test/scenarios` source tree, in a structure that roughly mirrors Metabase’s URL structure. For example, tests for the admin “datamodel” pages are located in `e2e/test/scenarios/admin/datamodel`.
 
-During development you will want to run `yarn build-hot` to continuously build the frontend, and `yarn test-cypress-open` to open the Cypress application where you can execute the tests you are working on.
+### Standard Development Flow
+1. Run `yarn build-hot` to continuously build the frontend
+
+2. then `yarn test-cypress-open --browser=electron` to open the Cypress application where you can execute tests you are working on
```

Comment:
we added `--browser=electron` mostly because chrome crashes randomly starting from version ~114

Reply:
@uladzimirdev @oleggromov Specifying a browser in the CLI mostly makes sense for `cypress run`, not for `cypress open` because in the open mode Cypress will detect ALL available browsers and will let you choose from the GUI. This was unneeded addition, imo.